adapter/pkg/config/log_types.go

```diff
@@ -25,7 +25,11 @@ type pkg struct {
 type accessLog struct {
 	Enable  bool
 	LogFile string
-	Format  string
+	// ReservedLogFormat is reserved for Choreo Gateway Access Logs Observability feature. Changes to this may
+	// break the functionality in the observability feature.
+	ReservedLogFormat string
```

Comment:
Do we need something like if,
SecondaryLogFormat is available then pick it and combine with ReservedLogFormat
If Format is available discard Reserved Log Format and use what is mentioned under Format.
If both are not available, just proceed with ReservedLogFormat ?

Reply:
We agreed with the existing approach.